django_freeradius/base/models.py

```diff
@@ -511,3 +517,60 @@ class Meta:
 
     def __str__(self):
         return str(self.username)
+
+
+class AbstractRadiusBatch(TimeStampedEditableModel):
+    users = models.ManyToManyField(settings.AUTH_USER_MODEL,
+                                   blank=True,
+                                   related_name='radius_batch')
+    file = models.FileField(upload_to=settings.MEDIA_ROOT,
```

Comment:
which file is this? The CSV or the PDF? I think we need two fields for both. The CSV can be uploaded only the first time and then is readonly. The PDF will be autogenerated only when a CSV is not supplied. Didn't we say to do this?